src/Core/src/Handlers/HybridWebView/HybridWebViewHandler.Windows.cs

```diff
@@ -33,11 +34,16 @@ protected override async void ConnectHandler(WebView2 platformView)
 				platformView.Loaded += OnWebViewLoaded;
 			}
 
-			platformView.CoreWebView2.Settings.AreDevToolsEnabled = true;//EnableWebDevTools;
-			platformView.CoreWebView2.Settings.IsWebMessageEnabled = true;
-			platformView.CoreWebView2.AddWebResourceRequestedFilter($"{AppOrigin}*", CoreWebView2WebResourceContext.All);
+			PlatformView.DispatcherQueue.TryEnqueue(async () =>
```

Comment:
There are now a few places in the code where the Windows code that uses WebView2 does a "ensure WebView2 is initialized, and *then* execute" pattern. This is because WebView2 is the only webview we use that has an async pattern to ensure it is ready. For that reason in places where the code is synchronous but needs WebView2 to be ready, it dispatches the work item to an async task, and awaits ready-ness, and then does what it wants. This is the same pattern we did in BlazorWebView.